src/components/Requirements/Requirements.vue

```diff
@@ -107,13 +118,13 @@ type Data = {
   showAllCourses: ShowAllCourses;
   shouldShowAllCourses: boolean;
   showAllPage: number;
+  tourStep: number;
 };
 
 // This section will be revisited when we try to make first-time tooltips
-const tour = introJs().start();
+const tour: any = introJs().start();
```

Comment:
why we do need any here

Reply:
I mentioned this in the last bullet of "Remaining TODOs" above, but I was getting the following error...

`IntroJS: Property '_currentStep' does not exist on type 'IntroJs'.`

Per @tcho6319's suggestion, I was about to remove this `any` type by using `require` instead of `import` syntax for my newest commit, but I am not sure if this is better? Let me know what you think.

Reply:
Using require simply suppresses the problem, which is almost same as `any`...
I think it might help to bump introjs and its typedef. In recent 3.0 release, you can get the current step more easily by `introjs().currentStep()`: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48969/files